{"status": "passed", "labels": [{"name": "story", "value": "账户、密码输入错误用例"}, {"name": "feature", "value": "登录用例"}, {"name": "tag", "value": "run(order=2)"}, {"name": "tag", "value": "@pytest.mark.usefixtures('init_driver', 'setup')"}, {"name": "tag", "value": "smoke"}, {"name": "parentSuite", "value": "webtest.testcases"}, {"name": "suite", "value": "test_loginpytest"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "cnki-PC"}, {"name": "thread", "value": "632648-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "webtest.testcases.test_loginpytest"}], "parameters": [{"name": "data", "value": "{'expected': '帐号或密码错误!', 'pwd': 'python1', 'phone': '18684720553'}"}], "fullName": "webtest.testcases.test_loginpytest.TestLogin#test_login_2", "name": "test_login_2[data1]", "stop": 1594457984406, "historyId": "d61d8550cf1e25dc7f3a7782fbb558ca", "start": 1594457983341, "testCaseId": "4de8035ea60fcdf87d07e0a66f2d92ca", "uuid": "43570817-9189-4b85-bf49-175ea45b1be2"}